### DISCUSSION TITLE
I can't find some events.

### DISCUSSION THREAD

**Discussion by ZakolkaTV** [EDCD/EDDI#2596] [EDDI VoiceAttack Q&A] [answered]
Hi guys. Thank you so much for your incredible work.

I found various events related to stations for VoiceAttack. For example, (Docking granted event), (Docking denied event), (Station no fire zone entered event) and others.

But I couldn't find the events:
- when the pilot blocks another landing pad;
- when the pilot blocks the entrance/exit area of the station;
- when the pilot flies into the station without prior request for docking.

If these events are not present, then can they be added to EDDI? It's important to me.

**Comment by Tkael** [comment]
There are events when you receive fines for these actions. But I assume you're asking for an event for before you receive a fine? 
I'm not sure the journal contains enough information for us to generate events at that time.

**Comment by ZakolkaTV** [comment, marked as answer]
I found the answer to my question.
What I needed is in the event ((EDDI message received)). The station tells us what we are doing wrong, and I can use the station's responses as events of the pilot's actions.

**Comment by ZakolkaTV** [comment]
@Tkael 
Please tell me, is there any event or variable that can determine if the target was darkened before the jump?

**Reply by Tkael** [reply]
I'm not sure what you mean by "darkened". If you mean obscured by a planet then I don't believe such a variable or event exists.
